app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+<resources>
+    <string name="app_name">FOrder</string>
+    <string name="mgs_product_time_out">Sản phẩm đã quá giờ đặt hàng, bạn vui lòng chọn sản phẩm khác!</string>
+    <string name="mgs_this_order">Bạn thực sự muốn đặt giỏ hàng này?</string>
+    <string name="mgs_order_all">Bạn muốn đặt hàng tất cả?</string>
+    <string name="mgs_delete_product">Bạn có muốn xóa sản phẩm này?</string>
+    <string name="mgs_accept_order">Bạn thật sự muốn đóng đơn hàng?</string>
+    <string name="action_yes">Có</string>
+    <string name="action_no">Không</string>
+    <string name="OK">Đồng ý</string>
+    <string name="action_order_all">Đặt hàng tất cả</string>
+    <string name="title_total">Tổng cộng :\u0020 </string>
+    <string name="title_tab_home">Trang chủ</string>
+    <string name="title_tab_search">Tìm kiếm</string>
+    <string name="title_tab_nitofy">Thông báo</string>
+    <string name="title_tab_profile">Cá nhân</string>
+    <string name="action_see_more">Xem tất cả</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.action_see_more` appears to be unused
 Unused resources make applications larger and slow down builds.